.github/workflows/build.yml

```diff
@@ -31,8 +32,23 @@ jobs:
       
       - name: Test the extension
         run: |
-          python -m pip install --upgrade -v -e ".[test, examples, docs]"
+          pip install --upgrade -v -e ".[test, examples, docs]"
           yarn run lint:check
 
           pytest
           yarn run test
+
+      - name: Build wheel
+        run: python -m build
+
+      - uses: actions/upload-artifact@v3
+        with:
+          name: dist
+          path: dist/
```

Comment:
Do we want to update the whole `dist` folder? Right now it contains the wheel, a tar file, and also the js files:
```
(.venv) ➜  deephaven-ipywidgets git:(main) ✗ ls -la dist
total 936
drwxr-xr-x   6 bender  staff     192 Aug  4 13:01 .
drwxr-xr-x  43 bender  staff    1376 Aug  4 12:59 ..
-rw-r--r--   1 bender  staff  206110 Aug  4 13:01 deephaven_ipywidgets-0.1.0.dev0-py2.py3-none-any.whl
-rw-r--r--   1 bender  staff  139461 Aug  4 12:59 deephaven_ipywidgets-0.1.0.dev0.tar.gz
-rw-r--r--   1 bender  staff   25321 Aug  4 12:56 index.js
-rw-r--r--   1 bender  staff   94824 Aug  4 12:56 index.js.map
```

Reply:
We should also update the README publishing instructions (these were from cookiecutter): https://github.com/deephaven/deephaven-ipywidgets#releasing-your-initial-packages

Not sure what `twine` is or if we should bother to use it for publishing.

Reply:
Well, these are for GH artifacts only - arguably, the index.js stuff may be useful for debugging. For PyPi releases, we want the wheel and the source tar.gz. I'm not sure if PyPi will upload the index.js b/c it's in the dist folder - but arguably, if it's not something we want to upload, we probably shouldn't be putting it in the dist/ folder...

Reply:
Okay, yea we _could_ put the JS files somewhere else... I see they've already got some `.npmignore` stuff to remove the python build output, and the JS package doesn't have that... and wouldn't it just be the wheel going to PyPi instead of the whole dist folder? So we should be good.

Side note - just realizing the JS package is named `deephaven-ipywidgets` instead of `@deephaven/ipywidgets`... I should probably rename it to the latter (so it appears under the `@deephaven` scope/organization).

Reply:
I'm renaming the JS package as part of https://github.com/deephaven/deephaven-ipywidgets/pull/5